extensions/amp-a4a/0.1/amp-a4a.js

```diff
@@ -1136,10 +1149,11 @@ export class AmpA4A extends AMP.BaseElement {
   /**
    * Gets the Ad URL to send an XHR Request to.  To be implemented
    * by network.
+   * @param {?CONSENT_POLICY_STATE} unusedConsentState
    * @param {Promise<!Array<rtcResponseDef>>=} opt_rtcResponsesPromise
    * @return {!Promise<string>|string}
    */
-  getAdUrl(opt_rtcResponsesPromise) {
+  getAdUrl(unusedConsentState, opt_rtcResponsesPromise) {
```

Comment:
just to confirm only doubleclick use `opt_rtcResponsesPromise`?

Reply:
Currently yes but any other FF network could elect to include RTC support